nilearn/image/tests/test_resampling.py

```diff
@@ -281,27 +286,15 @@ def test_resampling_error_checks():
             interpolation="an_invalid_interpolation",
         )
 
-    # Resampling a binary image with continuous or
-    # linear interpolation should raise a warning.
-    data_binary = rng.randint(4, size=(1, 4, 4), dtype="int32")
-    data_binary[data_binary > 0] = 1
-    assert sorted(list(np.unique(data_binary))) == [0, 1]
-
-    rot = rotation(0, np.pi / 4)
-    img_binary = Nifti1Image(data_binary, np.eye(4))
-    assert _utils.niimg._is_binary_niimg(img_binary)
 
-    with pytest.warns(Warning, match="Resampling binary images with"):
-        resample_img(img_binary, target_affine=rot, interpolation="continuous")
-
-    with pytest.warns(Warning, match="Resampling binary images with"):
-        resample_img(img_binary, target_affine=rot, interpolation="linear")
-    img_no_sform = Nifti1Image(data, affine)
-    img_no_sform.set_sform(None)
-    with pytest.warns(Warning, match="The provided image has no sform"):
-        resample_img(img_no_sform, target_affine=affine)
+def test_resampling_noop():
```

Comment:
the name is not completely clear about what the test does (I still think that a short docstring helps).

Reply:
tried to refactor and improve the name and doc string